src/Index.ts

```diff
@@ -15,15 +17,58 @@ app.use(express.urlencoded({ extended: true }));
 // This is necessary to transfer data in a local environment
 app.use(cors());
 
+const db = new DatabaseConnection();
+db.connect();
+
+function checkFields(body: Object, res: any, ...fields: string[]): boolean {
+    const missing: string[] = [];
+
+    fields.forEach(field => {
```

Comment:
In the long run there are libraries that might be able to let us validate request schemas without writing custom code. There's something called class-validator for TypeScript I came across the other day that looks kind of cool and might let us use decorators to define the schema we're expecting in the model instead of needing to write our own validation code.

What you have works for this PR but I think cross-validator or something similar might be worth considering in the future.